src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,73 @@
+package wolox.training.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Try to avoid using *.

Reply:
👍 Follow this to configure intellij https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-